kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/JobOperationsImpl.java

```diff
@@ -94,6 +94,8 @@ private void waitUntilJobIsScaled() {
       public void run() {
         Job job = getMandatory();
         atomicJob.set(job);
+        if (job.getStatus().getActive() == null)
```

Comment:
I'd rather not mutate the job object here - can you extract `getActive` to an Integer variable & zero it if null? Then use that in the comparison below?

And please use curlies around blocks please.